pythonfuzz/tracer.py

```diff
@@ -0,0 +1,23 @@
+import collections
+import sys
+
+prev_line = 0
+data = collections.defaultdict(set)
+
+def trace(frame, event, arg):
+    if event != 'line':
+        return trace
+    global prev_line
+    func_filename = frame.f_code.co_filename
+    func_line_no = frame.f_lineno
+
+    data[func_filename].add((prev_line, func_line_no))
+    prev_line = func_line_no
+    return trace
+
+def get_coverage() -> int:
```

Comment:
annotations aren't supported in python 2, so if the python 2 backport is merged, this will fail.